.github/workflows/github-actions.yml

```diff
@@ -2,28 +2,32 @@ name: Selenium CI
 
 on:
   push:
-    branches: [ master ]
+    branches: [ master, main ]
   pull_request:
     branches: [ master ]
 
 jobs:
   build:
 
     runs-on: windows-latest
+    strategy:
+      fail-fast: false
```

Comment:
why?

Reply:
In order to continue the execution in case of fails to detect other possible failures